keras_core/backend/tensorflow/nn.py

```diff
@@ -420,6 +420,14 @@ def one_hot(x, num_classes, axis=-1, dtype="float32"):
     return tf.one_hot(x, num_classes, axis=axis, dtype=dtype)
 
 
+def multi_hot(x, num_classes, axis=-1, dtype='float32'):
```

Comment:
If we're going to add new backend functions, they should have a corresponding Operation class and function in `keras_core/ops/nn.py`.`multi_hot` is useful, so it's worth doing for that one. I think `count` is likely to be confusing though since it doesn't follow the semantics of https://numpy.org/doc/stable/reference/generated/numpy.ma.count.html

Reply:
I added MultiHot to `keras_core/ops/nn.py`.

Regarding the `count` function, what do you recommend in order to follow the semantics of `numpy.ma.count`? Should we change the name of the `count` function in our implementation to make it different from `numpy.ma.count`?

Reply:
Just move the non-public functions to categorical_encoding as side functions. They don't need to be in the backend at all. And implement them using `ops` so they're backend agnostic.

Reply:
I found that it might not work if I implemented any function using `ops` because from what I understood is that while importing the backend it will be imported based on the `KERAS_BACKEND` which won't change again if we changed the backend inside `TFDataLayer.__call__` function, so I used `self.backend` instead of using `ops`, what do you think?